command_line/export_best.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import absolute_import, division, print_function
+
+import logging
+import sys
+
+from libtbx.phil import parse
+from libtbx import Auto
+
+logger = logging.getLogger("dials.command_line.export_best")
+
+help_message = """
+This program is used to export the results of dials processing for the strategy
+program BEST.
+
+"""
+
+phil_scope = parse(
+    """
+
+  intensity = *auto profile sum scale
+    .type = choice(multi=True)
+    .help = "Choice of which intensities to export. Allowed combinations:
+            scale, profile, sum, profile+sum, sum+profile+scale. Auto will
+            default to scale or profile+sum depending on if the data are scaled."
+
+  debug = False
+    .type = bool
+    .help = "Output additional debugging information"
+
+  best {
+
+    prefix = best
+      .type = str
+      .help = "The prefix for the output file names for best"
+              "(.hkl, .dat and .par files)"
+
+    n_bins = 100
+      .type = int(value_min=1)
+      .help = "Number of resolution bins for background estimation"
+
+    min_partiality = 0.1
+      .type = float(value_min=0, value_max=1)
+      .help = "Minimum partiality of reflections to export"
+
+  }
+
+  output {
+
+    log = dials.export_best.log
+      .type = path
+      .help = "The log filename"
+
+    debug_log = dials.export_best.debug.log
```

Comment:
I thought we were no longer creating debug.log files? (see https://github.com/dials/dials/issues/832)

Reply:
We are; have now removed this in both `export` and `export_best`